Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Create_or_Edit_EDL_object.yml

```diff
@@ -0,0 +1,528 @@
+      script: '|||prisma-sase-external-dynamic-list-update'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      folder:
+        simple: ${inputs.Folder}
+      frequency:
+        simple: ${inputs.Frequency}
+      id:
+        complex:
+          root: PrismaSase.ExternalDynamicList
+          accessor: id
+      overwrite:
+        simple: ${inputs.Overwrite}
+      predefined_ip_list:
+        simple: ${inputs.IPListName}
+      predefined_url_list:
+        simple: ${inputs.URLListName}
+      source_url:
+        complex:
+          root: inputs.SourceURL
+      tsg_id:
+        complex:
+          root: inputs.TSGID
+      type:
+        complex:
+          root: inputs.EDLObjectType
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 660,
+          "y": -120
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+system: true
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 1275,
+        "width": 800,
+        "x": 240,
+        "y": -630
+      }
+    }
+  }
+inputs:
+- key: Name
+  value: {}
+  required: false
+  description: The external dynamic list object name.
+  playbookInputQuery:
+- key: Folder
+  value:
+    simple: Shared
+  required: false
+  description: |-
+    The configuration folder group setting.
+    The default value is 'Shared'.
+  playbookInputQuery:
+- key: TSGID
+  value: {}
+  required: false
+  description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+  playbookInputQuery:
+- key: IPListName
+  value: {}
+  required: false
+  description: |-
+    The predefined IP list. If the type is predefined_ip, a value must be provided.
+    Possible values: panw-bulletproof-ip-list,panw-highrisk-ip-list,panw-known-ip-list,panw-torexit-ip-list.
+  playbookInputQuery:
+- key: URLListName
+  value: {}
+  required: false
+  description: |-
+    The predefined URL list. If the type is predefined_url, a value must be provided.
+
+    Possible value:
+    panw–auth-portal-exclude-list
+  playbookInputQuery:
+- key: SourceURL
+  value: {}
+  required: false
+  description: The source URL. If the type is ip, url or domain, a value must be provided.
+  playbookInputQuery:
+- key: Frequency
+  value: {}
+  required: false
+  description: |-
+    Frequency to check for updates on the EDL URL.
+    Possible values:
+    daily,five_minute,hourly,monthly,weekly
+  playbookInputQuery:
+- key: FrequencyTime
+  value: {}
+  required: false
+  description: The frequency hour. If the frequency argument is daily, weekly or monthly, a value must be provided. Possible values are 00-23.
+  playbookInputQuery:
+- key: EDLObjectType
```

Comment:
Type is a mandatory input for the command, we should check if the type isnt empty

Reply:
Set to mandatory:`true`

Reply:
for the command 'prisma-sase-external-dynamic-list-create' - type is mandatory 
thats why you need to add a check if type isn't empty 
![image](https://github.com/demisto/content/assets/61933087/365cefde-ac3c-48b1-9971-089a36ee9fd1)

Reply:
Added a manual step to verify if the Type argument isn't empty.
![image](https://github.com/demisto/content/assets/88268646/0b30da25-e4cc-48ee-866b-935629cd6c26)